packages/Babylonian-Tests.package/BPPrintbuggerTest.class/instance/testUpdatesSwimlanesAfterDeactivatingExample.st

```diff
@@ -4,6 +4,5 @@ testUpdatesSwimlanesAfterDeactivatingExample
 	self initExamplesWitMethodAWithAssertionUsingExpressionResult.
 	self assert: printbugger canvasMorph submorphs size equals: 2.
```

Comment:
😬 `#assert:equals:` works the other way round... This is not ideal, but unfortunately, the default. So it is `assert: expected equals: actual`